public/config/configuration.json

```diff
@@ -14,7 +14,7 @@
   "NAMEREQUEST_LD_CLIENT_ID": "5fbec4b5d998e30990020007",
   "LEGAL_API_URL": "https://legal-api-dev.apps.silver.devops.gov.bc.ca",
   "LEGAL_API_VERSION": "/api/v1",
-  "NAMEX_API_URL": "https://namex-dev.apps.silver.devops.gov.bc.ca",
+  "NAMEX_API_URL": "https://namex-api-dev.apps.silver.devops.gov.bc.ca",
```

Comment:
Just trying to run this branch locally and the requests to the url are failing. I'm guessing Namex-Api will require a new build to Dev before this will take effect?

Reply:
oops thank you haha. I have been hitting the api locally, but was just trying this and thought I had an auth issue

Reply:
oh right on! I will update locally and give it another try :)